games_services/shared_apple/Player.swift

```diff
@@ -17,6 +17,26 @@ class Player: BaseGamesServices {
     let gamePlayerAlias = currentPlayer.alias
     result(gamePlayerAlias)
   }
+
+  func isUnderage(result: @escaping FlutterResult) {
+    result(currentPlayer.isUnderage)
+  }
+
+  func isMultiplayerGamingRestricted(result: @escaping FlutterResult) {
+    if #available(iOS 13.0, *) {
+      result(currentPlayer.isMultiplayerGamingRestricted)
+    } else {
+      result(nil)
+    }
+  }
+
+  func isPersonalizedCommunicationRestricted(result: @escaping FlutterResult) {
+    if #available(iOS 14.0, *) {
+      result(currentPlayer.isPersonalizedCommunicationRestricted)
+    } else {
+      result(nil)
```

Comment:
Same here,
Can you return this result in case iOS version is less than 13.0
```
result(PluginError.notSupportedForThisOSVersion.flutterError())
```

Reply:
And done.